Wabbajack.App.Wpf/View Models/Installers/InstallerVM.cs

```diff
@@ -271,8 +288,32 @@ public InstallerVM(ILogger<InstallerVM> logger, DTOSerializer dtos, SettingsMana
                         .Where(t => t.Failed)
                         .Concat(Validate())
                         .ToArray();
-                    if (!errors.Any()) return ErrorResponse.Success;
-                    return ErrorResponse.Fail(string.Join("\n", errors.Select(e => e.Reason)));
+                    if (errors.Length == 0)
+                    {
+                        UnrecognisedFilesPresent = false;
+                        return ErrorResponse.Success;
+                    } else
```

Comment:
Try to stick to the standard C# syntax that WJ uses everywhere, so the colons are on separate lines